api_tests/btsieve/test.js

```diff
@@ -7,12 +7,22 @@ const actor = require("../lib/actor.js");
 const ethereum = require("../lib/ethereum.js");
 const wallet = require("../lib/wallet.js");
 const btsieve_conf = require("../lib/btsieve.js");
-
-const bitcoin_rpc_client = bitcoin.create_client();
+const utils = require("web3-utils");
+
+// TODO: Do not expose BitcoinRpcClient
```

Comment:
:question: Is this a personal TODO or meant to be done afterwards?

Reply:
Meant to be done afterwards as it's easier done if the `happy.js` are TypeScript.